mlflow/models/evaluation/base.py

```diff
@@ -122,6 +122,7 @@ def __init__(
         version=None,
         metric_details=None,
         metric_metadata=None,
+        custom_metric_config=None,
```

Comment:
Can we rename `custom_metric_config` to represent more concrete usage, like `make_genai_metric_fn_args`? The terms `metric_details` / `metric_metadata` / `custom_metric_config` sound all the same - vaguely represent some sort of metadata.

Reply:
good idea. but I'll genai_metric_args to shorten the name